theme/snippets/home-header.liquid

```diff
@@ -0,0 +1,13 @@
+<div class="HomeHeader serif-xl site-padding-x pt_625 md:pt1_25 md:pb5">
+  Library is a community space and forum for exchange. Our 
+  <a href="/offerings" class="text-decoration-none color-black" target="_blank" rel="noreferrer noopener" aria-label="{{ 'snippets.home_header.links.offerings_link.aria_label' | t }}">
```

Comment:
Do we have a global `_reset.scss` file? I will usually reset all `a` tags in there, that way we don't have to add `text-decoration-none color-black` to every a tag :/. I also like to set the default color to `currentColor` that way the link will inherit its color from its closest parent.

Reply:
Ahh you just saved me a lot of time typing that for every `a` tag! And first time I'm hearing about `currentColor`! Thanks!!